ekip/game/templates/adventure/choose_adventure.html

```diff
@@ -24,19 +24,19 @@
         <ul>
           <li>
             <a href="{% url 'adventure_nature_walk' %}">
-            <img src="{% static "img/activity/nature-walk.jpg" %}" alt="nature walk" />
+            <img src="{% static "img/activity/nature-walk.jpg" %}" alt="I decided to go on a nature walk." />
             <button>Go on a nature walk!</button>
             </a>
           </li>
           <li>
             <a href="{% url 'adventure_time_travel' %}">
-            <img src="{% static "img/activity/time-travel.jpg" %}" alt="time travel" />
+            <img src="{% static "img/activity/time-travel.jpg" %}" alt="I decided to time travel." />
             <button>Time travel!</button>
             </a>
           </li>
           <li>
             <a href="{% url 'adventure_swimming' %}">
-            <img src="{% static "img/activity/beach.jpg" %}" alt="beach" />
+            <img src="{% static "img/activity/beach.jpg" %}" alt="I decided to go swimming." />
```

Comment:
Do you mean it should just say "go swimming" ? Because the page on the whole reads "I decided to go swimming"